lib/ansible/content.rb

```diff
@@ -1,5 +1,7 @@
 module Ansible
   class Content
+    PLUGIN_CONTENT_DIR = Rails.root.join("content", "ansible").to_s.freeze
```

Comment:
I'm not sure about this directory because we always intended that the content directory of manageiq core could also be pluggable (that is, it could bring its own content things as well).  We might want another directory that's not `/content`

Reply:
Yeah, I'm fine with putting this somewhere else, but I wanted it to be in the source tree so that we could get at it when running locally.

That said, even if there were playbooks here as you describe, the behavior wouldn't really be any different right? We would then just have to copy those from here to the new location anyway so I don't really see any harm in keeping this.